internal/api/stats/api/query_test.go

```diff
@@ -0,0 +1,114 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/db"
+	"github.com/edulinq/autograder/internal/stats"
+	"github.com/edulinq/autograder/internal/timestamp"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+func TestQuery(test *testing.T) {
+	db.ResetForTesting()
+	defer db.ResetForTesting()
+
+	testCases := []struct {
```

Comment:
There will need to be more query-focused test cases.
See: internal/api/courses/stats/query_test.go